game_event/templates/game-events.html

```diff
@@ -49,6 +52,14 @@ <h1>Game Events</h1>
 				<td>{{ event.min_number_of_players }}</td>
 				<td>{{ event.max_number_of_players }}</td>
 				<td><a href="http://127.0.0.1:8000/courts/{{ event.court.courtID }}">{{ event.court.courtID }}</a></td>
```

Comment:
I know you didn't change this but this is bad it will not work in a "real" deployment.
you can change this to `/courts/` and I think the browser will be able to use the base url

Reply:
@SchSeba Fixed.